src/tailwind.config.js

```diff
@@ -80,6 +80,7 @@ module.exports = {
       10: '10',
       'flash-messages': 'var(--z-index-flash-messages)',
       overlay: 'var(--z-index-overlay)',
+      'overlay-loader': 'var(--z-index-overlay-loader)',
```

Comment:
for completeness, I do want to point out that I'd like to see the overlay loader totally removed. (It's a bit jarring).
But that needs to be a whole project (figuring out loading / intermediate states, etc)

Reply:
I don't see `--z-index-flash-messages` (or any of these variables) anywhere in a `:root`, is that something users of this library need to provide in their own css?

Reply:
> yup -- we currently define all of those in our internal core ui library's addon.css.

That's kind of weird combined with creating the classes here. Should we just move them into `themes.json` or some other config file here?

Even with Tailwind 3, `z-[var(--z-index-overlay-loader)]` feels pretty unwieldy and hard to discover (not to mention, not autocomplete or linting?)

> Something like

Would it need to be:

```css
'overlay-loader': 'calc(var(--z-index-overlay) + 10px)';
```

?

Reply:
I think z-index is just a number - no `px` required. 

Had to look it up to confirm:
https://developer.mozilla.org/en-US/docs/Web/CSS/z-index

Reply:
> Should we just move them into themes.json or some other config file here?

Sounds reasonable - in Tailwind 3, they'd all be opt-in, anyway

> feels pretty unwieldy and hard to discover (not to mention, not autocomplete or linting?)

yeah having autocomplete is pretty compelling :thinking: